packages/frontend-shared/src/gql-components/OpenConfigFileInIDE.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <template v-if="query?.data?.value">
-    <button
-      data-testid="open-config-file"
+    <div
```

Comment:
We tend to render another `button` element within this component to modify styling, resulting in buttons in buttons and invalid markup. A simple div works here as the click events with bubble through, but it's not the best.

Reply:
Change reverted, good call.